charts/sysdig-deploy/README.md

```diff
@@ -65,6 +65,39 @@ helm install -n sysdig-agent sysdig sysdig/sysdig-deploy -f values.sysdig.yaml
 
 Further configuration information can be found below.
 
+## Migrating from `sysdig`
+
+To easily migrate from the previous `sysdig` chart to the new unified `sysdig-deploy` chart, use the migration helper script from this repo.
```

Comment:
would be good to add 2 things:
1. mention the move from helm 2 to helm 3
2. explain what the script does: reformat the values.yaml value to preserve values from old chart or something like that

Reply:
added!